jekyll/_cci2/workflows.md

```diff
@@ -7,11 +7,11 @@ categories: [configuring-jobs]
 order: 30
 ---
 
-This section describes the Workflows feature and provides examples for parallel, sequential, fan-in, fan-out, and branch-level workflows. To enable Workflows, you must first split the single job in your `config.yml` file into multiple jobs, if you have not already done so. Job names must be unique within a `config.yml` file. 
+This section describes the Workflows feature and provides examples for parallel, sequential, fan-in, fan-out, and branch-level workflows. To enable Workflows, you must first split the single job in your CircleCI 2.0 `config.yml` file into multiple jobs with unique names, if you have not already done so. Job names must be unique within a `config.yml` file. See [Migrating from 1.0 to 2.0]({{ site.baseurl }}/2.0/migrating-from-1-2/) for instructions.
```

Comment:
Right, I qualify it by saying 'if you have not already done so' for that reason. We will have both, but I haven't heard of this case you describe with using workflows for a single job, we can add that later based on feedback and usage.